object-cache.php

```diff
@@ -7,6 +7,15 @@
  * can use this to guarantee uniqueness for the keys generated by this object cache.
  */
 
+if ( ! defined( 'WP_REDIS_USE_RELAY' ) ) {
+	define( 'WP_REDIS_USE_RELAY', false );
+}
+
+// Conditionally run the script if: the correct class exists, and the script has not been disabled.
+if ( ! class_exists( WP_REDIS_USE_RELAY ? 'Relay\Relay' : 'Redis' ) && ( defined( 'WP_REDIS_DISABLED' ) && WP_REDIS_DISABLED ) ) {
```

Comment:
Should this have an || in the middle?

`if ( ! class_exists( WP_REDIS_USE_RELAY ? 'Relay\Relay' : 'Redis' ) || ( defined( 'WP_REDIS_DISABLED' ) && WP_REDIS_DISABLED ) ) {`

In other words, if the class doesn't exist **or** WP Redis is disabled, then return.